openpype/hosts/nuke/plugins/publish/extract_review_baking_streams.py

```diff
@@ -25,6 +25,34 @@ class ExtractReviewDataMov(publish.Extractor):
     viewer_lut_raw = None
     outputs = {}
 
+    @classmethod
+    def apply_settings(cls, project_settings):
+        """just in case there are some old presets
+        in deprecrated ExtractReviewDataMov Plugins
+        """
+        nuke_publish = project_settings["nuke"]["publish"]
+        deprecrated_review_settings = nuke_publish["ExtractReviewDataMov"]
+        current_review_settings = (
+            nuke_publish["ExtractReviewDataBakingStreams"]
+        )
+        if deprecrated_review_settings["viewer_lut_raw"] == (
+            current_review_settings["viewer_lut_raw"]
+        ):
+            cls.viewer_lut_raw = (
+                current_review_settings["viewer_lut_raw"]
+            )
+        else:
+            cls.viewer_lut_raw = (
+                deprecrated_review_settings["viewer_lut_raw"]
+            )
+
+        if deprecrated_review_settings["outputs"] == (
+            current_review_settings["outputs"]
+        ):
+            cls.outputs = current_review_settings["outputs"]
+        else:
+            cls.outputs = deprecrated_review_settings["outputs"]
```

Comment:
there is a attribute `enabled` we could use `project_settings/nuke/publish/ExtractReviewDataMov/enabled` So if it is enabled then it is having priority on `ExtractReviewDataMov` and once you disable it it would prioritize `ExtractReviewDataBakingStreams`